documentation/MeetingMinutes/2024/MOM-2024-02-14.md

```diff
@@ -0,0 +1,106 @@
+# Identity and Consent Management meeting
+
+#### *February 14, 2024*
+
+<br>
+
+Preliminary Minutes of Meeting. To be updated during and after the meeting.
```

Comment:
Next time, you might want to remove this "preliminary" comment when the final meeting minutes are ready, right? (It is not important anyway)